.travis.yml

```diff
@@ -13,30 +13,6 @@ env:
 
 jobs:
   include:
-    - python: "3.7"
-      env:
-        GDALVERSION="3.0.4"
-        PROJVERSION="6.2.1"
-    - python: "3.7"
-      env:
-        GDALVERSION="3.1.4"
-        PROJVERSION="6.3.2"
-    - python: "3.8"
-      env:
-        GDALVERSION="3.1.4"
-        PROJVERSION="7.0.1"
-    - python: "3.8"
-      env:
-        GDALVERSION="3.2.3"
-        PROJVERSION="7.1.1"
-    - python: "3.9"
-      env:
-        GDALVERSION="3.2.3"
-        PROJVERSION="7.2.1"
-    - python: "3.8"
-      env:
-        GDALVERSION="3.3.2"
-        PROJVERSION="8.0.1"
```

Comment:
@snowman2 what do you think about deleting this file? We're not using Travis CI anymore.

Reply:
I had that thought as well. I agree that it can be removed. Updated the PR.